community/kernel/src/main/java/org/neo4j/kernel/api/exceptions/index/IndexEntryConflictException.java

```diff
@@ -61,23 +63,32 @@ public RuntimeException notAllowed( NewIndexDescriptor descriptor )
                 descriptor.userDescription( SchemaUtil.idTokenNameLookup ) ), this );
     }
 
-    public String evidenceMessage( String labelName, String propertyKey )
+    public String evidenceMessage( TokenNameLookup tokenNameLookup, LabelSchemaDescriptor schema )
     {
+        assert schema.getPropertyIds().length == propertyValues.values().length;
```

Comment:
since we already have a specific value object for property values i think it's super strange that we fall back to backing array to get that would propose to have propertyValues.size() method instead